docs/release-notes/release-1-4-0.md

```diff
@@ -0,0 +1,69 @@
+# Release Notes - ISLE v.1.3.0, 2019-09-27
```

Comment:
# Release Notes - ISLE v.1.4.0, 2019-11-25

Reply:
Just fixed this.Thanks!